components/frontend/src/utils.js

```diff
@@ -193,28 +202,39 @@ sortWithLocaleCompare.propTypes = {
     strings: stringsPropType,
 }
 
+function hideMetric(metric, metricsToHide, hiddenTags) {
+    const hideBecauseNoActionNeeded =
+        metricsToHide === "no_action_needed" && ["target_met", "debt_target_met", "informative"].includes(metric.status)
+    const hideBecauseNoIssues = metricsToHide === "no_issues" && !metric.issue_ids
+    const hideBecauseTagIsHidden =
+        hiddenTags?.length > 0 &&
+        hiddenTags?.filter((hiddenTag) => metric.tags?.includes(hiddenTag)).length >= metric.tags?.length
```

Comment:
Okay, so filtering tags is additive and not combinative. Think it makes sense, perhaps good to document or tooltip?

Reply:
It works quite intuitively in the UI I think. Not sure it's necessary to try and explain it to the users.